src/go/rdctl/cmd/snapshotCreate.go

```diff
@@ -38,7 +44,17 @@ func createSnapshot(args []string) error {
 		return nil
 	}
 
-	if _, err := manager.Create(name, snapshotDescription); err != nil {
+	// Ideally we would not use the deprecated syscall package,
+	// but it works well with all expected scenarios and allows us
+	// to avoid platform-specific signal handling code.
+	ctx, _ := signal.NotifyContext(context.Background(), syscall.SIGINT, syscall.SIGHUP, syscall.SIGTERM)
```

Comment:
We should keep the `stop` function returned, and do a `defer stop()` to make sure it's called if we're done creating the snapshot without the signal handler ever being called.